graphql_compiler/macros/macro_edge/directives.py

```diff
@@ -68,8 +69,19 @@
 
 def get_schema_for_macro_edge_definitions(querying_schema):
     """Given a schema object used for querying, create a schema used for macro edge definitions."""
-    new_directives = list(chain(
-        querying_schema.get_directives(), DIRECTIVES_REQUIRED_IN_MACRO_EDGE_DEFINITION))
+    original_directives = querying_schema.get_directives()
+    check_for_nondefault_directive_names(original_directives)
+
+    names_of_allowed_directives = {
+        directive.name for directive in
```

Comment:
nit: move the `for` to the next line for consistency with the list comprehension on lines 80-84.